consensus/src/testing/mod.rs

```diff
@@ -65,18 +65,20 @@ pub fn gen_config(node_ix: NodeIndex, n_members: NodeCount) -> Config {
     }
 }
 
+type HonestMemeber = (
```

Comment:
Same comment as above, although if it turns out that in this case it's a lot of work around very unrelated files, then maybe it can be skipped.

Reply:
Eh, nvm, 4 objects are probably not worth it. Leaving comment in case you disagree.